closed/OpenJ9.gmk

```diff
@@ -453,15 +459,34 @@ ifneq (,$(OPENJ9_DEVELOPER_DIR))
   CUSTOM_COMPILER_ENV_VARS += DEVELOPER_DIR="$(OPENJ9_DEVELOPER_DIR)"
 endif
 
+
+ifeq (true,$(OPENJ9_ENABLE_CMAKE))
+$(OUTPUTDIR)/vm/cmake.stamp:
```

Comment:
Please remove the added blank line above - we don't need sequences of multiple blank lines.
Please retain the spacing around `:`.
What would trigger a second execution of this rule (when something changes that would necessitate running cmake again)?

Reply:
As long as the build tree isnt deleted, cmake shouldn't need to be re-run manually. As part of the generated makefiles, it will auto re-run itself as required

Reply:
Changing some `CMakeLists.txt` files will likely invalidate the work done by the first invocation of `cmake`: is the only recourse to `make clean` and start over? I would hope we can do better.

Reply:
in the makefiles that cmake generates, it adds a rule to re-run cmake if  any of the cmakelists.txt have been modified